sdks/wasm/framework/src/System.Net.Http.WebAssemblyMessageHandler/RequestCache.cs

```diff
@@ -0,0 +1,44 @@
+// Copyright (c) .NET Foundation. All rights reserved.
```

Comment:
Please don't add any binaries to the repo sdks/wasm/framework/System.Net.Http.WebAssemblyMessageHandler.0.2.2.nupkg